src/components/homePage/homePage.jsx

```diff
@@ -8,28 +9,35 @@ import TableContainer from '@material-ui/core/TableContainer';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
 
 import { memberActions } from '../../state/actions';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
     table: {
       minWidth: 650,
     },
-});
+    root: {
+        '& > *': {
+          margin: theme.spacing(1),
+        },
+    },
+}));
 
 function HomePage() {
     const members = useSelector(state => state.members);
     const dispatch = useDispatch();
 
     useEffect(() => {
-        dispatch(memberActions.getAll());
+        dispatch(memberActions.getAllMember());
     }, [dispatch]);
 
     const classes = useStyles();
 
     return (
         <div className="col-lg-12">
-            <h3>All registered members:</h3>
+            <div>All registered members:
+            <span><Link to="/member/register" style={{float: "right"}}  className="btn btn-primary">Add new member</Link></span></div>
```

Comment:
So the Link here provided by react-router-dom can be replaced with something like:
`<Link component={RouterLink} to="/member/register">`
The RouterLink here can be imported as such:
`import { Link as RouterLink } from "react-router-dom";`
This way we will be using both functionality of react-router-dom Link and style and uniformity of using Link from material-ui.